src/main/java/net/refsonyak/endgameaddons/Config.java

```diff
@@ -0,0 +1,5 @@
+public class Config {
+    /*Class that contains all configurable variables for quick'n'easy balance changes
```

Comment:
Please word more carefully and format the comment to either use `//` or have `/*` and `*/` on a separate line.